security.php

```diff
@@ -54,56 +57,202 @@ function wpcom_vip_is_restricted_username( $username ) {
 		|| WPCOM_VIP_MACHINE_USER_EMAIL === $username;
 }
 
+/**
+ * PRIVATE: This function is not intended to be used outside of this file and is subject to change without notice.
+ *
+ * Returns the defualts for limits.
```

Comment:
```suggestion
 * Return the defaults for login and password reset authorization limits, windows, and lockout periods.
 *
 * @internal The values are subject to change without notice.
```

ref: https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/internal.html#internal